db/crud.go

```diff
@@ -258,10 +258,6 @@ func (db *Database) getRev(docid, revid string, maxHistory int, historyFrom []st
 		return DocumentRevision{}, err
 	}
 
-	if revision.BodyBytes == nil {
```

Comment:
If we want to avoid regressions, I think we'd want to leave this intact, and just switch to 
if revision.BodyBytes == nil && !revision.Removal ?

Reply:
Sure, this change is included in the latest commit.